src/RealMeService.php

```diff
@@ -416,7 +416,7 @@ public static function currentRealMeUser()
      * @param HTTPRequest $request
      * @param string $backUrl
      * @return bool|null true if the user is correctly authenticated, false if there was an error with login
-     * @throws OneLogin_Saml2_Error
+     * @throws Error
```

Comment:
This is a breaking change

Reply:
so I guess it's good to target `master` after all ;)
I'm not fussed with this being a major. It is what it is to be semver compliant.
I don't see another way around this other than doing some kind of class alias in a `_config.php` or suchlike I guess.

Reply:
You could add `OneLogin_Saml2_Error` exceptions that extend the new ones, mark them as deprecated, and then this could be a minor change

Reply:
I feel like this is excessive in order to simply avoid changing a number. I think the impact of not providing exact BC compatibility is quite low for this module.

Reply:
I agree, but it's "semantic" for a reason =D let's just do a major version bump

Reply:
Neat, sounds good (and we're not introducing debt) \o/